code-orange/MainViewController.swift

```diff
@@ -137,11 +142,21 @@ class MainViewController: UIViewController {
     view.setAutoLayoutHeight(1)
     return view
   }()
+
+  private lazy var drawerView: DrawerView = {
+    let drawerView = DrawerView()
+    drawerView.translatesAutoresizingMaskIntoConstraints = false
+    return drawerView
+  }()
   
   override func viewDidLoad() {
     super.viewDidLoad()
     view.addSubview(mainStack)
-    mainStack.pin(to: view, anchors: [.leading(0), .trailing(0), .top(28), .bottom(0)])
+    mainStack.pin(to: view, anchors: [.leading(0), .trailing(0), .top(28), .bottom(-24)])
```

Comment:
still hacky move of bottom rounded corners below the display
This also means the map is 24 pixels below the bottom of the screen, thus hiding the google logo. We can fix that later